src/olympia/blocklist/tests/test_admin.py

```diff
@@ -490,6 +490,8 @@ def _test_add_multiple_submit(self, addon_adu):
         content = response.content.decode('utf-8')
         # meta data for new blocks and existing ones needing update:
         assert 'Add-on GUIDs (one per line)' not in content
+        total_adu = addon_adu + addon_adu - 1
```

Comment:
mm, where does that come from?

Reply:
That's being defined a few lines above, line 458 and 471

Reply:
I suggest we introduce variables for these then, like `self.new_addon_adu`.

Reply:
d40eb14dc3